src/scss/grommet-core/_objects.article.scss

```diff
@@ -58,9 +58,13 @@
 .article__control {
   position: fixed;
   z-index: 10;
+  // using margin instead of padding for article control
+  // to prevent arrow from overlapping scrollbar and
+  // hindering scrollbar usage
+  margin: $inuit-base-spacing-unit;
 
   .button__icon {
-    padding: $inuit-base-spacing-unit;
+    padding: 0;
```

Comment:
Previously the hit target was 96x96. After this PR, the hit target will be 48x48.

Before:
![screen shot 2016-05-31 at 7 25 50 am](https://cloud.githubusercontent.com/assets/10161095/15684038/f6dc9398-2700-11e6-97ce-d11c2a0b2b10.png)

After (48x48 hit target):
![screen shot 2016-05-31 at 7 23 33 am](https://cloud.githubusercontent.com/assets/10161095/15684037/f35bdfbc-2700-11e6-8c8b-a568e1ab0601.png)